docs/dev/libvirt/README.md

```diff
@@ -151,6 +151,22 @@ so we only need to allow traffic on the libvirt port:
 sudo firewall-cmd --zone=libvirt --add-service=libvirt
 ```
 
+*NOTE:* The installer creates a libvirt network and a `tt0` bridge.
+This bridge name can be customized in the `install-config.yaml` file in the
+`platform.libvirt.network.if` section as:
```

Comment:
Not sure if it's a great idea if we want to tell people to modify the template in the source. Rather they should generate the install-config and then modify it.

Reply:
That's what I wanted to say, to modify the install-config.yaml file and include the platform.libvirt.network.if section

Reply:
Ah, i misread, sorry. However, i see that it's assuming reader already know about install-config.yaml file. Adding a link to `docs/user/overview.md#multiple-invocations` would be more than enough.

Reply:
Done. Thanks